CHANGELOG.md

```diff
@@ -9,6 +9,7 @@
 * Add support for _Kafka node pools_ according to [Strimzi Proposal #50](https://github.com/strimzi/proposals/blob/main/050-Kafka-Node-Pools.md)
 * Update OpenTelemetry 1.19.0
 * Fixed ordering of JVM performance options [#8579](https://github.com/strimzi/strimzi-kafka-operator/issues/8579)
+* Upgrade Vert.x to 4.4.3
```

Comment:
I probably wouldn't mention it here. It is just an internal implementation detail without any meaning to users.

Reply:
I had doubt about it, I saw that we put this in some versions but not others :-)
Anyway I can remove, yes.

Reply:
We have warnings which make the build failing because of some deprecations, I will take a look.